scripts/forecast.py

```diff
@@ -76,6 +76,17 @@ def run_forecast(
         grouping_factors,
     )
 
+    cumulative_projections = cumulative_projections.with_columns(
+        pl.Series(
+            "estimate",
+            cumulative_projections.select(
+                [col for col in cumulative_projections.columns if "estimate_" in col]
+            )
+            .to_numpy()
+            .mean(1),
```

Comment:
```suggestion
            )
            .mean()
            .to_numpy()
```
Personally I find it easier to think in polars-land, rather than in both polars and in numpy

Reply:
Forthcoming updates switch to long format.